apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/subsection/SubSectionCreator.java

```diff
@@ -37,6 +44,35 @@ public CreateSubSectionResponse createSubSection(long pageId, Long userId, Creat
         }
     }
 
+
+    public DeleteSubSectionResponse deleteSubSection(DeleteSubSectionRequest request) {
+        try {
+            log.info("Deleting section");
+            Integer orderNbr = waUiMetaDataRepository.getOrderNumber(request.subSectionId());
+            waUiMetaDataRepository.deleteById(request.subSectionId());
+            waUiMetaDataRepository.updateOrderNumberByDecreasing(orderNbr, request.subSectionId());
+            return new DeleteSubSectionResponse(request.subSectionId(), "Sub Section Deleted Successfully");
```

Comment:
I believe the same logic applies here as above, you cannot delete a subsection if it has any content.

so I think the check would be "is the wa_ui_metadata entry with the next order number: "
1. null
2. another subsection
3. another section
4. a tab

if one of those 4 are true, then allow the delete.